website/templates/components/dashboard_templates.mako

```diff
@@ -153,7 +153,7 @@
             <br />
             <label>Template (Optional)</label>
             <span class="help-block">Start typing to search. Selecting project as template will duplicate its structure in the new project without importing the content of that project.</span>
-            <input type="hidden" class="select2-container createNodeTemplates" style="width: 100%">
+            <input type="hidden" class="select2-container createNodeTemplates" id="createNodeTemplatesInput" style="width: 100%">
```

Comment:
@lyndsysimon Yes,  `createNodeTemplates` is used in previous PR.  However, you and I had discussed before that we have to use a class to select the DOM, instead of ID. I think it is fine to remain both class and id.